regulations/templates/regulations/chrome.html

```diff
@@ -33,6 +33,7 @@
 {% include "regulations/main-header.html" %}
 
 <div class="sub-head {% block sub-head-class %}{% endblock %}">
+    {% block sub-head-drawer-icons %}
     <a href="#content-wrapper" class="hidden-text">Skip to main content</a> <!-- skip to content link for screen readers-->
```

Comment:
Hmmm, in our case, we're emptying the block and moving the drawer icons somewhere else. In our case it makes sense to move the skip nav along with the drawer icons. I think it's okay to leave the skip-nav within the block, but maybe a) we should pick a better block name or b) put the skip-nav in it's own block?

IMHO I think it's okay to leave as is.